src/steps/10-FinancialDetails/IGCE/CannotProceed.vue

```diff
@@ -1,15 +1,73 @@
 <template>
-  <div>
-    Future page for Cannot Proceed
-  </div>
+  <v-container fluid class="container-max-width mb-7">
+    <v-row>
+      <v-col class="col-12">
+        <h1 class="page-header mb-3">
+          To accurately complete your price estimate, you need to revisit the
+          Contract Details and Performance Requirements sections
```

Comment:
AC 1 - need logic to show different ends to the title here based on what's missing

Reply:
done.